### ISSUE TITLE
repo variable does not return anything when using groups

### ISSUE THREAD

**Issue by etfeet** [ytti/oxidized#395]
I have a post store script that uploads a tmos binary ucs export to git when there is a config change which depends on the OX_REPO_NAME variable. However, when I setup oxidize to use groups and per group git repositories, I am no longer having the repository returned from the repo name variable.

node=$OX_NODE_NAME
repo=$(echo $OX_REPO_NAME | sed 's/^.*networking\///g' | sed 's/\.git//g')
commit=$OX_REPO_COMMITREF
echo $(date): node: $node repo: $repo commit: $commit file: $gitbase/$repo/$node.ucs
...

```
output: 
Wed Apr 13 17:45:05 PDT 2016: node: 1.1.1.1 repo: commit: ca9dff9a3e8700feeb40726c8167bf55acd08cc8 file: /etc/oxidized/git/networking//1.1.1.1.ucs

groups:
  f5:
    username: myuser
    password: mypass

  git:
    user: oxidized
    email: oxidized@example.com
    repo: "/var/opt/gitlab/git-data/repositories/networking/default.git"

  ucs_update:
    type: exec
    events: [post_store]
    cmd: '/bin/bash /etc/oxidized/scripts/f5-ucs.sh'
    async: true
    timeout: 120
```

**Comment by ytti** [comment]
pong @aakso

**Comment by ytti** [comment]
repong @aakso

**Comment by roedie** [comment]
This is the same issue as #334

**Comment by ytti** [comment]
Closed due to #503
